frontend/fingertips-frontend/app/chart/page.test.tsx

```diff
@@ -128,36 +125,17 @@ describe('Chart Page', () => {
     });
   });
 
-  describe('when a single parent area is available ', () => {
-    it('should make 1 call for get area data when there is one area selected', async () => {
+  describe('when a single group is selected ', () => {
```

Comment:
```suggestion
  describe('when a single group is selected', () => {
```

Again, super minor but you've removed whitespace above, so could do it again here?